src/lib.rs

```diff
@@ -121,3 +122,14 @@ pub fn create_bios_disk_image(
 
     Ok(())
 }
+
+/// Prepare a folder for use with booting over UEFI_PXE. The dhcp server should
+/// have the filename option set to `bootloader`.
```

Comment:
> The dhcp server should have the filename option set to `bootloader`.

Could you clarify what this means and why it is necessary?

Reply:
That's part of how PXE works under the hood, the PXE client asks the DHCP server where to boot from. Very quick history lesson:
DHCP is based on BOOTP. BOOTP was designed to allow diskless workstations to boot from the network. The server-sent Offer and ACK packets would have their "Boot file name" and a "Next server IP address" fields populated to point the client to the TFTP server and the bootloader file.
DHCP still has those fields, but nowadays most DHCP servers (e.g. in home network WiFi routers) are mostly used to give out dynamic IP addresses and DNS configuration and don't populate the "Boot file name" field because they don't care about PXE.
The reason I explicitly mention what the bootloader file name option should be set to is that there's no standardized/default value for this field. The correct value depends entirely on where the bootloader file is located on the TFTP server which happens to be in `bootloader` with the current implementation. We also have to do this in the tests: We set the `bootfile` option on the network device which causes QEMU's integrated DHCP server to set the "Boot file name" field in the response packets:
https://github.com/rust-osdev/bootloader/blob/f975de96c65d2c850edeae3e8b852f2cf78c14a6/tests/runner/src/lib.rs#L64-L67

Reply:
Thanks a lot for the detailed explanation!